docs/platforms/android/configuration/shared-environments.mdx

```diff
@@ -16,11 +16,12 @@ When setting up Sentry inside a library, the consuming app could use the Sentry
 
 </Note>
 
-In order to not conflict with other Sentry instances, you should use the `Hub` API to create a new instance of Sentry.
-The Hub API works the same way as the global Sentry instance, but it is not global and can be used within your component. If you want to capture uncaught exceptions, you can use the `UncaughtExceptionHandlerIntegration` to capture them. As this will capture all uncaught exceptions within an app, you should use the `BeforeSendCallback` to only accept events that are relevant for your SDK.
+In order to not conflict with other Sentry instances, you should use the `Scopes` API to create a new instance of Sentry.
+The Scopes API works the same way as the global Sentry instance, but it is not global and can be used within your component. If you want to capture uncaught exceptions, you can use the `UncaughtExceptionHandlerIntegration` to capture them. As this will capture all uncaught exceptions within an app, you should use the `BeforeSendCallback` to only accept events that are relevant for your SDK.
 
 ```kotlin
-import io.sentry.Hub
+import io.sentry.IScope
```

Comment:
I think `IScope` should be replaced with `Scope` as that is used in the sample below:
```suggestion
import io.sentry.Scope
```